src/main/pages/setup-docker/docker-cli.adoc

```diff
@@ -152,127 +226,126 @@ These variables can be set in your local environment shell before running and th
+====
 
-'''''
+*_rmi_*
 
-_rmi_
+The `rmi` command deletes the Docker images from the local registry that Che has downloaded for this version.
 
-Deletes the Docker images from the local registry that Che has downloaded for this version.
+*_ssh_*
 
-'''''
+The `ssh` command connects the current terminal where the command is started to the terminal of a machine of the workspace. If no machine is specified in the command, it connects to the default development machine. The syntax is `ssh <workspace-name> [machine-name]`. The ssh connection works only if there is a workspace ssh key setup. A default ssh key is automatically generated when a workspace is created.
 
-_ssh_
+*_sync_*
 
-Connects the current terminal where the command is started to the terminal of a machine of the workspace. If no machine is specified in the command, it will connect to the default machine which is the dev machine. The syntax is `ssh <workspace-name> [machine-name]` The ssh connection will work only if there is a workspace ssh key setup. A default ssh key is automatically generated when a workspace is created.
+The `sync` command synchronizes contents of a workspace with a local directory mounted to `:/sync`. The syntax is `-v <path-on-your-machine>:/sync eclipse/che sync <workspace-name>`.
 
-'''''
+To display a log of the underlying unison tool, use the `--unison-verbose` flag.
 
-_sync_
+*_start_*
 
-Synchronizes a workspace’s contents with a local folder mounted to :/sync The syntax is `-v <path-on-your-machine>:/sync eclipse/che sync <workspace-name>`
+The `start` command starts Che and its services using the `docker-compose` command. If the system cannot find a valid configuration, it performs an `init` command. Every `start`command and `restart` command runs a `config` command to generate a new configuration set using the latest configuration. The starting sequence tests if any ports required by Che are currently being used by other services and to verify access to key APIs.
 
-To get extra information, the flag `--unison-verbose` can be used to display log of the underlying unison tool.
+To skip these checks, use the `--skip:preflight` and `--skip:postflight` command options. The typical Che start sequence includes an invocation of the `config` method, which regenerates configuration files placed into the `/instance` directory. To skip this generation, use the `--skip:config` command option. To automatically print out the server logs to the console during the booting of the server, append the  `--follow` argument to the command. To interrupt the output, press `CTRL-C to`.
 
-'''''
+*_stop_*
 
-_start_
+The default stop is a graceful stop where each workspace is stopped and confirmed shutdown before stopping system services. If workspaces are configured to snap on stop, all snaps are completed before the system service shutdown begins. You can ignore workspace stop behavior and shut down only system services using the `–force` flag.
```

Comment:
"confirmed shutdown" should be "confirmed shut down".

Basically, "shutdown" is the action of shutting something down, "shut down" is the state of _being_ shut down.